lib/data_structures/linked_list/circular_linked_list.rb

```diff
@@ -39,4 +39,13 @@ def pop
       head.previous_node.next_node = head
     end
   end
+
+  def find(value)
+    head.value = value
```

Comment:
Here my sentinel is setting head value as the value to be found.
If we reach the end of the circular linked list, it will return to the very beginning.